### PR TITLE
CORE-20655 Increasing kong.unirest client log level to debug issue

### PR DESCRIPTION
**Pull request by ben-millar**

### CODE REVIEW

**Comment by ben-millar** [comment]
build-e2e

**Comment by ben-millar** [comment]
build e2e

**Comment by ben-millar** [comment]
build-e2e

File: tools/corda-runtime-gradle-plugin/src/endToEndTest/resources/config/log4j2.xml [hunk 1]
```diff
@@ -44,6 +44,9 @@
         <!-- default to warn only for OSGi logging -->
         <Logger name="net.corda.osgi.framework.OSGiFrameworkWrap" level="warn" />
 
+        <!-- log debug for these 3rd party libs -->
+        <Logger name="kong.unirest" level="debug"/>
```
Comment (vkolomeyko) [inline tools/corda-runtime-gradle-plugin/src/endToEndTest/resources/config/log4j2.xml:48]: I think we might need to change this file:
https://github.com/corda/corda-e2e-tests/blob/release/5.3/e2etest/src/e2eTest/resources/log4j2-test.xml#L9
to enable UniRest logging at callsite for E2E JVM.

Comment (ben-millar) [reply tools/corda-runtime-gradle-plugin/src/endToEndTest/resources/config/log4j2.xml:48]: Thanks Viktor! Checking this out now

**Pull request closed without merging**